app/exercises/five.py

```diff
@@ -0,0 +1,104 @@
+from exercises.base import BaseTab
+from structures.weighted_adjacency_list import WeightedDirectedAdjacencyList
+
+import tkinter as tk
+from matplotlib.backends.backend_tkagg import FigureCanvasTkAgg, NavigationToolbar2Tk
+from matplotlib.figure import Figure
+import networkx as nx
+from tkinter import ttk, messagebox
+
```

Comment:
Keep order
import ...
import ...
from ...
from ...

Reply:
imports are in alphabetical order

Reply:
But should not